core-build/gulp-core-build-sass/package-lock.json

```diff
@@ -0,0 +1,10913 @@
+{
```

Comment:
This is a Rush repo. Delete this file.

You should install dependencies by running `rush install`.

Reply:
Got it, thank you.